doc/sphinx/source/vulkan.rst

```diff
@@ -1,20 +1,46 @@
 Vulkan driver
 =================
 
+NOTE: THIS DRIVER IS INCOMPLETE AND IS NOT ACTIVELY DEVELOPED BY TUNI.
```

Comment:
:D Perhaps just "This driver is incomplete without an active maintainer. Pull requests welcomed."